v2/devdoc/log_sink_etw_requirements.md

```diff
@@ -0,0 +1,179 @@
+# `log_sink_etw` requirements
+
+`log_sink_etw` implements a log sink interface that logs the events as ETW events.
+
+## References
+
+[Event Tracing For Windows](https://learn.microsoft.com/en-us/windows-hardware/drivers/devtest/event-tracing-for-windows--etw-)
+
+## Exposed API
+
+```
+    extern const LOG_SINK_IF log_sink_etw;
+```
+
+### log_sink_etw.log_sink_log
+
+The signature of `log_sink_etw.log_sink_log` is:
+
+```c
+typedef void (*LOG_SINK_LOG_FUNC)(LOG_LEVEL log_level, LOG_CONTEXT_HANDLE log_context, const char* file, const char* func, int line, const char* message_format, ...);
+```
+
+`log_sink_etw.log_sink_log` implements logging to an ETW provider.
```

Comment:
>to an ETW provider. [](http://example.com/codeflow?start=47&length=19)

in ETW language, I think "logging to" is "to a session". 

So maybe here is "logging as an ETW provider" ?

Reply:
Correct, fixed.